accessibility.md

```diff
@@ -29,6 +29,23 @@
   - Use the `button` element for JS interaction points and similar non-linking components. If a button is not of the `submit`, `reset`, or `menu` types, use `type="button"` to designate a generic button.
     - The `a` tag should only be used when using the `href` attribute to link "to other web pages, files, locations within the same page, email addresses, or any other URL" ([ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)).
 
+### HTML5 Elements
+
+  - HTML5 elements should be used whenever possible to communicate the element's role in the page. Some notable examples include:
+    - [`main`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) for the primary content of the page, most appropriately excluding repeated elements (e.g., sidebars, decorative marquees, site header and footer)
+    - [`article`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) for self-contained content that could conceivably belong in an RSS feed, such as blog posts, press releases, and event information (in both page and individually in index listings)
+    - [`section`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for thematically grouped content in a component that doesn't justify a stronger semantic element
+    - [`nav`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) for all significant navigation link lists
+    - [`aside`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside) for complementary content, such as advertising, "related article" lists, and repeated page sidebars
+  - Notably, using these roles not only serves visitors using accessibility technologies, but it also makes parsing code easier for fellow developers.
+
+**[⬆ back to top](#table-of-contents)**
+
+## ARIA and Element Roles
+
+  - In most cases, using semantic HTML elements makes explicit designation of element roles unnecessary (e.g., `role="presentation"`). The exceptions would be when they are serving slightly atypical roles.
```

Comment:
I think we want to replace "elements roles" with "landmark roles". A more explicit example of this would be `role="banner` to designate the header region of the page.

Reply:
Let's remove the last sentence or make it more explicit.